Tests/SwiftAlgorithmsTests/GroupedTests.swift

```diff
@@ -13,7 +13,7 @@ import XCTest
 import Algorithms
 
 final class GroupedTests: XCTestCase {
-  private class SampleError: Error {}
+  private final class SampleError: Error {}
```

Comment:
Can you please explain, why do we need this final keyword here?

Reply:
It was added to resolve the following error:
`Non-final class 'SampleError' cannot conform to 'Sendable'; use @unchecked Sendable`

Reply:
Thanks for explanation. Can you please also update you branch with base branch?